src/sc/sidechain.h

```diff
@@ -131,22 +140,29 @@ class CSidechain {
         READWRITE(balance);
         READWRITE(fixedParams);
         READWRITE(mImmatureAmounts);
+        READWRITE(scFees);
+        if (ser_action.ForRead())
+        {
+            if (!scFees.empty())
+                sizeOfScFeesContainers = scFees.size();
```

Comment:
`sizeOfScFeesContainers` should depend on the `getNumBlocksForScFeeCheck` and epoch length, not on `scFees.size`.
For example, `sizeOfScFeesContainers` was initialized  = 10, but the `scFees` consists of 3 elements (we had just a few epochs). After the node restart -> the value `sizeOfScFeesContainers == 3` instead of 10.

Reply:
We may even store `sizeOfScFeesContainers` value as well to avoid recalculation of that value, but can lead to the problem when we set the `-blocksforscfeecheck` flag on regtest to decrease the `sizeOfScFeesContainers`.